integration_test/api_end_to_end_test.go

```diff
@@ -199,6 +199,7 @@ func setupTestScenarioAndPublish(authOrgAdmin *httpexpect.Expect, authOrgViewer
 	triggerBody := `{
   "body": {
     "scoreReviewThreshold": 10,
+	"score_block_and_review_threshold": 10,
     "scoreRejectThreshold": 20,
```

Comment:
Probably use non deprecated prop name here ?

Reply:
Not sure what you mean - I'll change lines 201 and 203 in a follow-up PR (on removing the deprecated camel case dto fields)